core/client/src/test/java/alluxio/client/file/options/CreateDirectoryOptionsTest.java

```diff
@@ -45,14 +52,17 @@ public void fields() {
     Random random = new Random();
     boolean allowExists = random.nextBoolean();
     boolean recursive = random.nextBoolean();
+    Mode mode = new Mode((short) 0123);
```

Comment:
I generally agree with avoiding randomness, but for basic getter / setter / serialization / deserialization / equality tests I find randomness useful to improves coverage over time ... this is fine as is